### PR TITLE
Use docker, so I can build it and test it on windows

### PR DESCRIPTION
**Pull request by bruntonspall**
Just adds a simple dockerfile that can build and test the code, and run it easily.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from node:8
+label maintainer="michael@brunton-spall.co.uk"
+label uk.co.brunton-spall.version="0.0.1-beta"
```
Comment (timja) [inline Dockerfile:3]: what is this for?

File: README.md [hunk 2]
```diff
@@ -34,10 +34,28 @@ You'll also need a legacy API token, go get one from https://api.slack.com/custo
...
+
+```docker build -t xgovslackbot .
+```
+
+Run the built bot as is:
+```docker run -it -e "slackdomain=..." -e "token=xoxb-token" -e "apitoken=xoxp-apitoken" xgovslackbot nodejs ./index.js
+```
+
+Run the bot, but with local changes without rebuilding the base image:
+```docker run -it -e "slackdomain=..." -e "token=xoxb-token" -e "apitoken=xoxp-apitoken" --mount type=bind,source="$(pwd)/app",target=/usr/src/xgovslackbot/app xgovslackbot nodejs ./index.js
```
Comment (timja) [inline README.md:56]: this would be a lot easier with a simple `docker-compose` file ;)

File: Dockerfile (hunk 1 above)
Comment (bruntonspall) [reply Dockerfile:3]: I believe this is the replacement for the old MAINTAINER deprecation

Comment (timja) [reply Dockerfile:3]: the one above it is,
haven't seen anything like this line before,
also you can make it multiline to save layers in the image:
https://docs.docker.com/engine/reference/builder/#label

**Pull request merged**